polls/migrations/0006_experimenttype_experimenttypetaskpresentation_and_more.py

```diff
@@ -0,0 +1,100 @@
+# Generated by Django 4.1.1 on 2023-02-24 18:43
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("polls", "0005_alter_batch_created_at"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ExperimentType",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "type",
+                    models.CharField(
+                        choices=[("2AFC", "2AFC"), ("A/B", "A/B")], max_length=10
```

Comment:
This appears to create a DB constraint thus not allowing us to extend it dynamically by admin UI

Reply:
Ok so lets make it "type" of the Experiement a CharField with max_length=100. I guess that will do the job.

Reply:
Please